multibot.py

```diff
@@ -1,320 +1,318 @@
-import random
```

Comment:
Please rework this change. The whole code has been replaced - can you redo it and only but your changes into it?